caddyhttp/errors/setup.go

```diff
@@ -123,6 +123,10 @@ func errorsParse(c *caddy.Controller) (*ErrorHandler, error) {
 			}
 		}
 
+		if len(args) > 1 {
+			return handler, c.Errf("Only 1 Argument expected for errors directive")
+		}
```

Comment:
I think this should be lower case. `go lint` would've complained if this was part of `fmt.Errorf`.

Reply:
Yeah that was bad planning for convention on my part.